README.md

```diff
@@ -574,20 +574,38 @@ See also running on [cluster](doc/cluster.md), [YARN client](doc/yarn.md), on [E
 ### Manual steps
 
 1. Copy `config/local.sh.template` to `<environment>.sh` and edit as appropriate.  NOTE: be sure to set SPARK_VERSION if you need to compile against a different version.
-2. Copy `config/shiro.ini.template` to `shiro.ini` and edit as appropriate. NOTE: only required when `authentication = on`
-3. Copy `config/local.conf.template` to `<environment>.conf` and edit as appropriate.
-4. `bin/server_deploy.sh <environment>` -- this packages the job server along with config files and pushes
+2. If your `spark.submit.deployMode` is set `cluster` then update the following variables in bin/setenv.sh according to your environment.
+  - Cluster mode for spark-standalone
+```
+MANAGER_JAR_FILE="$REMOTE_JOBSERVER_DIR/spark-job-server.jar"
+MANAGER_CONF_FILE="$REMOTE_JOBSERVER_DIR/$(basename $conffile)"
+MANAGER_EXTRA_JAVA_OPTIONS=
+MANAGER_EXTRA_SPARK_CONFS="spark.yarn.submit.waitAppCompletion=false"
+MANAGER_LOGGING_OPTS="-Dlog4j.configuration=$REMOTE_JOBSERVER_DIR/log4j-cluster.properties"
+```
+
+  - Cluster mode for mesos/yarn
```

Comment:
I think it's better to have a separate example for each of mesos and yarn

Reply:
Ok. I will add separate examples.